hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/util/RestoreTool.java

```diff
@@ -169,6 +169,9 @@ public void incrementalRestoreTable(Connection conn, Path tableBackupPath, Path[
       for (int i = 0; i < tableNames.length; i++) {
         TableName tableName = tableNames[i];
         TableDescriptor tableDescriptor = getTableDescriptor(fileSys, tableName, incrBackupId);
+        if (tableDescriptor == null) {
+          throw new IOException("Can't found " + tableName + "'s descriptor ");
```

Comment:
Can't <strike>found</strike>... :arrow_right: Can't find...

Reply:
Oh,yes,some problems with grammar,I will modify it right away.